workspaces/ballerina/ballerina-extension/.env.example

```
@@ -1,4 +1,15 @@
+# Prod Copilot
 BALLERINA_ROOT_URL=https://dev-tools.wso2.com/ballerina-copilot
 BALLERINA_AUTH_ORG=
 BALLERINA_AUTH_CLIENT_ID=
 BALLERINA_AUTH_REDIRECT_URL=https://eae690d5-80c3-4fb7-9bc5-e8d747cca11b.e1-us-east-azure.choreoapps.dev
+BALLERINA_DEFAULT_COPLIOT_CODE_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BALLERINA_DEFAULT_COPLIOT_CODE_API_KEY key should go before the BALLERINA_ROOT_URL key

(UnorderedKey)